src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -55,11 +56,50 @@ public NodeSettings(Network innerNetwork = null, ProtocolVersion protocolVersion
             this.LoggerFactory.AddConsoleWithFilters();
             this.LoggerFactory.AddNLog();
             this.Logger = this.LoggerFactory.CreateLogger(typeof(NodeSettings).FullName);
+
+            // TODO: Temporary until Build method updated to call LoadConfiguration
+            loadConfiguration = true;
+
+            // Load arguments or configuration from .ctor?
+            if (args != null || loadConfiguration)
+                SetArgs(args);
+
+            // Load configuration from .ctor?
+            if (loadConfiguration)
+                this.LoadConfiguration();
+        }
+
+        /// <summary>
+        /// Sets the "command line" arguments.
+        /// </summary>
+        /// <param name="args">Arguments to set.</param>
+        private void SetArgs(string[] args)
+        {
+            this.LoadArgs = args ?? new string[] { };
```

Comment:
I'm still failing to see how the split has been done between SetArgs method and the LoadConfiguration method.
The LoadConfiguration method is still loading and reading the args.
Am I missing the point?

Reply:
Until we add the LoadConfiguration method to the next PR I do not expect anything to be happening differently at this point. The split was done as follows:
- Took the original LoadArguments method and moved the first few lines of that into the SetArgs method. The new LoadConfiguration method contains the remaining code of the LoadArguments method.